requirements.txt

```diff
@@ -1,6 +1,7 @@
 numpy~=1.21.4
 open3d~=0.14.1
 PyOpenGL~=3.1.5
-PyQt5~=5.14.1
+PyQt5~=5.15.7
 pytest~=7.1.1
-pytest-qt~=4.0.2
\ No newline at end of file
+pytest-qt~=4.0.2
+PyOpenGL-accelerate~=3.1.5
```

Comment:
To go into effect when installing with `pip` it also needs to be added here:
https://github.com/ch-sa/labelCloud/blob/master/setup.cfg#L51-L55

Reply:
[413e4b1](https://github.com/ch-sa/labelCloud/pull/97/commits/413e4b173faf1e7b6a49eb3eb5bb975b434e137c) added here.